extensions/arc/src/ui/components/radioOptionsGroup.ts

```diff
@@ -38,16 +40,17 @@ export class RadioOptionsGroup {
 				const radioOption = this._view!.modelBuilder.radioButton().withProperties<azdata.RadioButtonProperties>({
 					label: option,
 					checked: option === defaultValue,
+					name: this._groupName,
 					value: option,
 					enabled: true
 				}).component();
 				if (radioOption.checked) {
 					this._currentRadioOption = radioOption;
 				}
 				this._onNewDisposableCreated(radioOption.onDidClick(() => {
-					this._divContainer.items
-						.filter(otherOption => otherOption !== radioOption)
-						.forEach(otherOption => (otherOption as azdata.RadioButtonComponent).checked = false);
+					// this._divContainer.items
```

Comment:
remove

Reply:
You were somehow looking at an old iteration as this was already removed.